src/button/button.js

```diff
@@ -6,6 +6,7 @@ import StyledButton from './button.styled';
 
 const Button = ({ onClick, value, enabled, children, className }) => (
   <StyledButton
+    title={title}
```

Comment:
You missed adding `title` to the props destruction on line 7. As it is now, the game wont load.